emcc.py

```diff
@@ -1713,6 +1713,7 @@ def setup_pthreads(target):
     '__emscripten_tls_init',
     '_pthread_self',
     'checkMailbox',
+    '__emscripten_thread_mailbox_await',
```

Comment:
Move this up to a few lines do it is grouped with the other `__emscripten_thread_..` functions?